central/auth/m2m/kube_claim_extractor_test.go

```diff
@@ -0,0 +1,26 @@
+package m2m
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_kubeClaimExtractor(t *testing.T) {
```

Comment:
Although it would likely be similar to the generic extractor, would it make sense to have a test covering the successful claim extraction codepath ?